File: Benchmarks/normal/python/django/testcode/benchmark_test_12938.py
# SPDX-License-Identifier: Apache-2.0
from django.http import JsonResponse
import os
import importlib


def BenchmarkTest12938(request):
    env_value = os.environ.get('USER_INPUT', '')
    data = '%s' % (env_value,)
    importlib.import_module(str(data))
    return JsonResponse({"saved": True})

File: Benchmarks/normal/java/spring/testcode/BenchmarkTest12938.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest12938 {

    @GetMapping("/BenchmarkTest12938")
    public void BenchmarkTest12938(@RequestHeader("Origin") String origin, HttpServletRequest request, HttpServletResponse response) throws Exception {
        String originValue = origin != null ? origin : "";
        String data = "" + originValue;
        Integer.parseInt(data);
        response.setContentType("application/json");
        response.getWriter().print("{\"id\":0}");
    }
}
